<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Properties</class>
 <widget class="QDialog" name="Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="sizeInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="typeInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Размер</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Расположение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QLabel" name="pathInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
